README.md

```diff
@@ -39,10 +39,10 @@ Baza danych składa się z 6 tabel, każda z kolumn ma property not null:
     - user_id - INT - PK
     - points - INT - default 0
  4. **helper_rewards** - mapuje reakcje pod odpowiedziami na pytania do punktów którymi nagradzany jest odpowiadający;
-    - level - INT
-    - reaction_id - INT
+    - reward - INT - PK
+    - emoji_name - TEXT
```

Comment:
imo lepiej dać pk na emoji, dopuściłbym możliwość paru emotek z taką samą punktacją, ale na odwrót jest to niemożliwe

Reply:
jeśli chodzi o code extensibility to jak najbardziej, bo performance i tak nie ucierpi tutaj

Reply:
no to zmień tego pk i można mergować

Reply:
> some things in the repositories may not work, but they will be also changed after all and I suggest some changes in main.py when it comes to repositories initialization and imports.

a co miałeś tu na myśli? możesz sprecyzować?

Reply:
commands i settings wszędzie zostało takie samo ale tak jak przy przykładowo usłudze reaction role mamy tabelę dedykowaną tej usłudze => stary binding emoji : role z tabeli settings nie obowiązuje, więc jak robimy nasze poszczególne feature'y musimy pamiętać o ogarnięciu repository od danej usługi, taka uwaga zeby w razie czego wiedzieć gdzie muszą zostać wykonane zmiany przy pisaniu po przeoraniu bazy